drivers/hmis_csv_twenty_twenty_six/app/models/hmis_csv_twenty_twenty_six/importer/project.rb

```diff
@@ -79,6 +79,11 @@ def self.hmis_validations
             class: HmisCsvImporter::HmisCsvValidation::InclusionInSet,
             arguments: { valid_options: HudUtility2026.project_types.keys.map(&:to_s).freeze },
           },
+          # ProjectType can only be blank if ContinuumProject is not 1
+          {
+            class: HmisCsvImporter::HmisCsvValidation::NonBlankValidation,
+            arguments: { constraint_lambda: ->(item) { item['ContinuumProject'].to_s != '1' } },
```

Comment:
Is there a chance `item['ContinuumProject']` could be nil?
```suggestion
            arguments: { constraint_lambda: ->(item) { item['ContinuumProject']&.to_s != '1' } },
```

Reply:
Yes, but `nil.to_s` is just `""`, which isn't `'1'`, so it should be fine.